app/controllers/bids_controller.rb

```diff
@@ -0,0 +1,13 @@
+class BidsController < ApplicationController
+  include Secured
+
+  def create
+    user_id = current_user.id
+    auction_id = params[:id]
+    body = JSON.parse(request.body.read, symbolize_names: true)
```

Comment:
Try getting the request body off of the params object instead of parsing `request.body`